assignments/r-2.md

```diff
@@ -3,7 +3,18 @@ layout: page
 title: R Assignment 2
 subtitle: Vectors and Data Frames
 language: R
-exercises: ['Vectors 1', 'Vectors 2', 'Data Frames']
+exercises: ['Vectors 1', 'Vectors 2', 'Data Frames 1', 'Data Frames 2']
 ---
 
+####Learning Objectives
+
+> Following this assignment students should be able to:
+
+> - understand the vector and data frame object structures
+> - assign, subset, and manipulate data in a vector
+> - execute vector algebra
+> - import .CSV data files as a data frame
+
```

Comment:
Change the last bullet to "create data frames"

Add one more bullet:

* subset data from a data frame